code/controllers/subsystem/shuttles.dm

```diff
@@ -254,7 +254,7 @@ var/datum/subsystem/shuttle/SSshuttle
 	slip.ordernumber = ordernum
 
 	var/stationName = (errors & MANIFEST_ERROR_NAME) ? new_station_name() : station_name()
-	var/packagesAmt = SSshuttle.shoppinglist.len + ((errors & MANIFEST_ERROR_COUNT) ? 0 : rand(1,2))
```

Comment:
Upon further investigation, I don't see anything wrong. The only potential problem I see is https://github.com/tgstation/-tg-station/blob/master/code/controllers/subsystem/shuttles/supply.dm#L56 but I doubt that is happening.